shortcodes/sc-button.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Provides a shortcode for the .btn class.
+ *
+ * Note: button plugin features (toggle states, checkbox/radio buttons) are not
+ * supported by this shortcode.
+ **/
+if ( ! class_exists( 'ButtonSC' ) ) {
+	class ButtonSC extends ATHENA_SC_Shortcode {
+		public
+			$command = 'button',
+			$name = 'Button',
+			$desc = 'Creates a new link styled as an Athena button.',
+			$content = true;
+
+		/**
+		 * Returns the shortcode's fields.
+		 *
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 *
+		 * @return Array | The shortcode's fields.
+		 **/
+		public function fields() {
+			return array(
+				array(
+					'param'   => 'href',
+					'name'    => 'Button Link',
+					'type'    => 'text'
+				),
+				array(
+					'param'   => 'new_window',
+					'name'    => 'Open link in a new window',
+					'type'    => 'checkbox',
+					'default' => false
+				),
+				array(
+					'param'   => 'class',
+					'name'    => 'CSS Classes',
+					'desc'    => 'Separate each class with a single space. Refer to the Athena Framework documentation for available classes.',
+					'type'    => 'text'
+				),
+				array(
+					'param'   => 'id',
+					'name'    => 'CSS ID',
```

Comment:
CSS ID label seems strange to me since ID's are not specific to CSS, but if this is intended to be used only with CSS then I can see why the label would be CSS ID.